src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -227,6 +227,22 @@
   private static final String COMPACT_MAP_ENTRIES_DISPLAY = "Compact Map Entries";
   private static final boolean COMPACT_MAP_ENTRIES_DEFAULT = true;
 
+  public static final String DATA_STREAM_TIMESTAMP_CONFIG = "data.stream.timestamp";
```

Comment:
at this point we have 3 data stream configs, let's create their own separate config group like we do for kerberos and ssl